src/post.js

```diff
@@ -10,12 +10,42 @@ module.exports = async config => {
   const name = `v${pkg.version}`;
   const tag = {owner, repo, ref: `refs/tags/${name}`, sha: await promisify(gitHead)()};
   const body = await promisify(plugins.generateNotes)(config);
-  const release = {owner, repo, tag_name: name, name, target_commitish: branch, draft: !!debug, body};
+  let githubPreRelease = false;
+  let release = {
+    owner,
+    repo,
+    tag_name: name,
+    name,
+    target_commitish: branch,
+    draft: !!debug,
+    body,
+    prerelease: githubPreRelease,
+  };
 
   if (debug && !githubToken) {
     return {published: false, release};
   }
 
+  // obtain npm dist tag if any
+  let npmDistTag = (pkg.publishConfig || {}).tag;
```

Comment:
The tag can also be in the `.npmrc` config.
It actually already determined in [src/index.js#L54](https://github.com/semantic-release/semantic-release/blob/caribou/src/index.js#L54) and passed to `post.js` in `config.npm.tag` so you should reuse that.